### DISCUSSION TITLE
How far away is v2

### DISCUSSION THREAD

**Discussion by robcleghornCP** [flyerhq/flutter_chat_ui#584] [Q&A] [answered]
Thanks for building this package, very grateful for the contributions made. I see a few issues building up and mention of no more v1.x releases planned.  This puts a lot of us in a holding pattern, do you have any timelines for releasing v2.0?

**Comment by demchenkoalex** [comment]
To not repeat myself will post a reply from other issue: https://github.com/flyerhq/flutter_chat_ui/issues/577#issuecomment-2014436351

**Comment by ihadabs** [comment]
@demchenkoalex I appreciate your work! is there any update?

**Reply by demchenkoalex** [reply]
just finish a rough month at work with lots of tasks, coming back to the chat now. Hope will not see any blockers and will be able to publish at least v2 preview in the coming month.

**Comment by demchenkoalex** [comment, marked as answer]
v2 dev build is live https://github.com/flyerhq/flutter_chat_ui/issues/646
